locales/de/index.yml

```diff
@@ -1158,6 +1158,7 @@ wallet:
           bottomSheetTitle: "Warum kann ich nicht mit dieser Zahlungsmethode bezahlen?"
           bottomSheetDescription: "Die Funktion “In-App-Zahlungen” ist möglicherweise deaktiviert. Bevor du versuchst, die Zahlung erneut vorzunehmen, gehe bitte wie folgt vor:\n\n1. gehe zum Bereich “Konto”;\n2. wähle die gewünschte Zahlungsmethode aus;\n3. aktiviere “In-App-Zahlungen”."
```

Comment:
IIRC we shouldn't add DE locales unless they are fully translated, since they are fully optional. Are they necessary here?

Reply:
These localizations were already present, this PR adds nothing new.

Reply:
Removed in https://github.com/pagopa/io-app/pull/5291/commits/b586ddfac5f2ad49b6bd0f316741f29c64c3cd04